### ISSUE TITLE
Drawing problem on FMX with new Image32 3.4.1

### ISSUE THREAD

**Issue by carloBarazzetta** [EtheaDev/SVGIconImageList#205]
I've found some problems on FMX drawing after update to Image32 3.4.1
A strange "white" surround of images and some icons with white background instead of transparency.

![image](https://user-images.githubusercontent.com/7803548/139595306-8e3298bb-f140-48c0-8a83-885b8814f530.png)
![image](https://user-images.githubusercontent.com/7803548/139595328-a46a4569-eed1-47d9-8032-dc7bfd4006b3.png)

**Comment by AngusJohnson** [comment]
Hi Carlo.
This can be fixed by adding a line to TColorRenderer.RenderProc in Img32.Draw as demonstrated below:

```
procedure TColorRenderer.RenderProc(x1, x2, y: integer; alpha: PByte);
var
  i: integer;
  dst: PColor32;
begin
  dst := GetDstPixel(x1,y);
  for i := x1 to x2 do
  begin
    //BlendToAlpha is marginally slower than BlendToOpaque but it's used
    //here because it's universally applicable.
    //Ord() is used here because very old compilers define PByte as a PChar
    if Ord(alpha^) > 1 then // ADD THIS LINE
      dst^ := BlendToAlpha(dst^, ((Ord(alpha^) * fAlpha) shr 8) shl 24 or fColor);
    inc(dst); inc(alpha);
  end;
end;

```

**Comment by carloBarazzetta** [comment]
Fixed in 3.1.1 version
